### ISSUE TITLE
Location is not enabled after clicking 'Allow only while using this app' or 'Ask Everytime' in Android

### ISSUE THREAD

**Issue by vtvtvtvtvtvt** [FlutterFlow/flutterflow-issues#1672]
### Has your issue been reported?

- [X] I have searched the existing issues and confirm it has not been reported.
- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

![image](https://github.com/FlutterFlow/flutterflow-issues/assets/142770013/b1097513-a4d4-4104-bca5-84bbb22da5fe)

after setting up the action flow as seen in the image I am not getting options to allow or reject location access.

Note that I use to get the option before but it was not turning on location when I click on 'allow' and then raised it with Flutterflow support. After Support did something and asked me test, I am not getting the option to to allow or reject location access.

### Expected Behavior

when clicked on button with location permission defined as action, it should turn on location. and capture current device location.

### Steps to Reproduce

add a button
add action to button-> location permission
test it by clicking on button
It does not show pemissions to allow or deny.
if it shows the option to allow or deny it does not turn on location once clicked on 'Allow only while using this app' or 'Ask Everytime' .

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

ITFficmBy411rsdY1pSJbccwqCQvH08cTeUzlO1hdwwbI+LpBKQEYMnCRxBWYuqHewh+B1SJhn0x06GJkezxD8IpFy6WG7ZZzKlxUxDhUTiWRYinPqmWRWt5M8NMFUy117a32hJQXfhidVIM3zv3Fty+b3qCf9qOYwx5e6fDbOY=

### Context

unable to turn on location

### Visual documentation

![image](https://github.com/FlutterFlow/flutterflow-issues/assets/142770013/d294bf8f-2a0a-44d9-a601-1c9952a425f5)

### Environment

```markdown
- FlutterFlow version: 4.0
- Platform:andorid
- Browser name and version:
- Operating system and version affected:
```

**Comment by hariprasadms** [comment]
Hi @vtvtvtvtvtvt - Thanks for submitting the issue. The options you are expecting to see, you can see that only in real devices. There is no option in FF to enable or disable it. I would suggest that in the device you are trying to test it, clear permission access and try it again, it should work as expected.

**Comment by vtvtvtvtvtvt** [comment]
Hi @hariprasadms  - Let me rephrase this.  It does not turn on location once I click on 'Allow only while using this app' or 'Ask Everytime'. Attached is the Video recording for your reference,

**Comment by vtvtvtvtvtvt** [comment]
https://github.com/FlutterFlow/flutterflow-issues/assets/142770013/7c8c0fa7-54f3-45c0-b4d2-940c21feae0c

video attached

**Comment by vtvtvtvtvtvt** [comment]
any update on this?? and why is this tagged 'not a bug' it clearly is a bug if it does not turn on location.

**Comment by vtvtvtvtvtvt** [comment]
Any update??????????????????? This is pathetic support I have ever got for something I am paying.

**Comment by sbis04** [comment]
Hey @vtvtvtvtvtvt, that is an intended default behavior of location permission on Android devices. Just allowing permission doesn't mean it will start fetching the location, it will only get the location when you explicitly ask for a location like using the `PlacePicker` widget.
